### PR TITLE
Determine if the user can see the admin views

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Add a check to determine if the user can see the admin views

**Related issue(s)**

Review together with https://github.com/kyma-project/kyma/pull/2486

### CODE REVIEW

File: core/src/luigi-config/luigi-config.js [hunk 1]
```diff
@@ -419,6 +421,87 @@ function fetchFromGraphQL(query, variables) {
...
+    );
+  });
+}
+
+function navigationPermissionChecker(nodeToCheckPermissionsFor) {
+  let hasPermissions = false;
+  if (nodeToCheckPermissionsFor.adminOnly) {
+    if (selfSubjectRulesReview.length > 0) {
+      selfSubjectRulesReview.forEach(rule => {
+        if (
```
Comment (pekura) [inline core/src/luigi-config/luigi-config.js]: OK, this condition is.. not the final solution I suppose. But can't we do it better? Like having a list of api groups and verbs that we consider admin must have (as a constant) and a generic function that checks again any list of verbs and api groups (that would know that '*' means any etc..). We can talk to Philipp first to make sure that this approach with selfsubjectrules is, in general, not temporary.

File: core/src/luigi-config/luigi-config.js [hunk 2]
```diff
@@ -454,14 +537,24 @@ function relogin() {
...
+        modules.backendModules.forEach(backendModule => {
           backendModules.push(backendModule.name);
         });
       }
+      if (subjectRules && subjectRules.length > 0) {
+        selfSubjectRulesReview = subjectRules;
+      }
     },
     err => {
       console.error('Error while fetching backend modules', err);
```
Comment (pekura) [inline core/src/luigi-config/luigi-config.js]: Even if error cause is selfsubjectrules call the error message says:

luigiConfig.bundle.js:1 Error while fetching backend modules Forbidden (user=admin@kyma.cx, verb=create, resource=selfsubjectrulesreviews, subresource=).

File: core/src/luigi-config/luigi-config.js (hunk 1 above)
Comment (dariadomagala) [reply core/src/luigi-config/luigi-config.js]: No, it's not the final solution as described here: https://github.com/kyma-project/console/issues/459#issuecomment-458522570 . This check was actually a proposal from Philipp for the temporary solution.

Comment (pekura) [reply core/src/luigi-config/luigi-config.js]: Alright then!

**Pull request merged**